MIGRATING.MD

```diff
@@ -25,22 +27,35 @@ against the data you used to receive from the SignalFx Java Agent.
 Follow these steps to migrate from the SignalFx Java Agent to the Splunk
 distribution of Splunk Distribution of OpenTelemetry Java Instrumentation:
 
-1. Download the [latest release](https://github.com/signalfx/splunk-otel-java/releases)
-   of the Splunk Distribution of OpenTelemetry Java Instrumentation.
-2. Set the service name. This is how you can identify the service in APM.
-   You can set the service name with a system property or environment
-   variable. This is how you can set it with an environment variable with a
-   service name of `yourServiceName`:
+1. Download the [latest release](https://github.com/signalfx/splunk-otel-java/releases/latest/download/splunk-otel-javaagent-all.jar)
+   of the Splunk Distribution of OpenTelemetry Java Instrumentation. For example use: 
+   ```bash
+   $ # download the newest version of the agent
+   $ sudo curl -vsSL -o splunk-otel-javaagent-all.jar 'https://github.com/signalfx/splunk-otel-java/releases/latest/download/splunk-otel-javaagent-all.jar'
    ```
+2. Set the service name. This is how you can identify the service in Splunk APM.
+
+   Example how you can set it with an environment variable with a
+   service name of `yourServiceName` with environment variable:
```

Comment:
```suggestion
   For example, you can set the service name using an environment variable:
```

Reply:
It goes like "an example how to do it this... or this... way". So I prefer "This is an example" rather "for example". But will add an article as it is missing now (in both cases).